components/levelCompleted/LevelCompleted.tsx

```diff
@@ -1,7 +1,12 @@
+import { useEffect, useState } from "react";
 import styles from "./LevelCompleted.module.css";
 
 function LevelCompleted() {
-  const displayedScore = JSON.parse(localStorage.getItem("score"));
+  const [displayedScore, setDisplayedScore] = useState("");
+  useEffect(() => {
+    setDisplayedScore(JSON.parse(localStorage.getItem("score")));
+  });
```

Comment:
What happens if displayScore is not yet recorded? You might get an error because the LocalStorage is not available on the server side.

[look](What happens if displayScore is not yet recorded? You might get an error because the LocalStorage is not available on the server side.

https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Statements/try...catch)

```suggestion
  useEffect(() => {
  if (!displayScore) {
      ?
  }
    setDisplayedScore(JSON.parse(localStorage.getItem("score")));
  });
```

Reply:
In my opinion, the case is excluded by the fact that the level cannot be completed with zero points. What do you think?